README.rst

```diff
@@ -470,7 +470,7 @@ and IIS for Windows.
 
 * The HTTPD (HTTP Daemon) receives the request.
 * The server breaks down the request to the following parameters:
-   * HTTP Request Method (either ``GET``, ``HEAD``, ``POST``, ``PUT``,
+   * HTTP Request Method (either ``GET``, ``HEAD``, ``POST``, ``PUT``, ``PATCH``,
```

Comment:
Please wrap this line, it's longer than our maximum.

Reply:
PR updated